documentation/modules/rn-2.2.adoc

```diff
@@ -9,21 +9,29 @@ You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {vir
 
 The release notes describe new features and enhancements, known issues, and technical changes.
 
-// [id="technical-changes-22_{context}"]
-// == Technical changes
-//
-// .TBD
-//
-// TBD
-//
-// [id="new-features-and-enhancements-22_{context}"]
-// == New features and enhancements
-//
-// This release adds the following features and improvements.
-//
-// .TBD
-//
-// TBD
+[id="technical-changes-22_{context}"]
+== Technical changes
+
+.Setting the precopy interval for warm migration
+
+You can set the interval between snapshots taken during the precopy stage of warm migration.
```

Comment:
interval **time**

Reply:
@mguetta1 
But interval already means time (between events).

Reply:
Sasha, you are correct but Maayan's version is clearer. Please make the change. Thanks.

Reply:
@mguetta1
@apinnick 
Then maybe "time interval"?

Reply:
@sbeskin-redhat Yes. It is better

Reply:
That's fine. You do not need to ask us about such a small change.